content/fast/defer-unused-css/index.md

```diff
@@ -0,0 +1,150 @@
+
+* Extract the class definitions marked with green in the report obtained from the coverage tool, and put those classes inside a `<style>` block at the head of the page:
+
+```
+<style type="text/css">
+.accordion-btn {background-color: #ADD8E6;color: #444;cursor: pointer;padding: 18px;width: 100%;border: none;text-align: left;outline: none;font-size: 15px;transition: 0.4s;}.container {padding: 0 18px;display: none;background-color: white;overflow: hidden;}h1 {word-spacing: 5px;color: blue;font-weight: bold;text-align: center;}
+</style>
+```
+
+* Then, load the rest of the classes asynchronously, by applying the following pattern:
+
+```
+<link rel="preload" href="styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'"> 
+<noscript><link rel="stylesheet" href="styles.css"></noscript>
+```
+
+This is not the standard way of loading CSS. Here's how it works:
+
+* `link rel="preload" as="style"` requests the stylesheet asynchronously. You can learn more about `preload` in the [Preload critical assets guide](https://web.dev/fast/preload-critical-assets).
+* The `onload` attribute in the `link` allows the CSS to be processed when it finishes loading.
+* "nulling" the `onload` handler once it is used helps some browsers avoid re-calling the handler upon switching the rel attribute.
+* The reference to the stylesheet inside of a `noscript` element works as a fallback for browsers that don’t execute JavaScript.
+
+The [resulting page](https://defer-css-optimized.glitch.me/) looks exactly like the previous version, even when most of its styles are loaded asynchronously:
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    allow="geolocation; microphone; camera; midi; encrypted-media"
+    src="https://glitch.com/embed/#!/embed/defer-css-optimized?path=index.html&previewSize=100&attributionHidden=true"
+    alt="defer-css-optimized on Glitch"
+    style="height: 100%; width: 100%; border: 0;">
+  </iframe>
+</div>
+
+## Monitor
+
+Use the Devtools to run another **Performance** trace on the [optimized page](https://defer-css-optimized.glitch.me/). 
+
+The **FCP** marker appears before the page requests the CSS, which means the browser doesn’t need to wait for the CSS to load before rendering the page:
+
+<figure>
+  <img src="./cdt-perf-optimized.png"
+       alt="DevTools performance trace for unoptimized page, showing FCP starting before CSS loads." class="screenshot">
+</figure>
+
+As a final step, run Lighthouse on the optimized page. 
+
+In the report you’ll see that the FCP page has been reduced by **0.2s** (a 20% improvement!):
+
+<figure>
+  <img src="./lighthouse-metrics-optimized.png"
+       alt="Lighthouse report, showing an FCP value of '0.8s'." class="screenshot">
+</figure>
+
+Also, as a result, the **Eliminate render-blocking resources** suggestion is no longer under **Opportunities**, and now belongs to the **Passed Audits** section:
```

Comment:
Suggestion to lean this a little-- remove 'also as a result'... "The Eliminate render-blocking resources suggestion is no longer under Opportunities, and now belongs in the Passed Audits section: